ggshield/core/config/config.py

```diff
@@ -130,12 +135,24 @@ def api_url(self) -> str:
     @property
     def dashboard_url(self) -> str:
         """
-        The dashboard URL to use to use the dashboard
+        The dashboard URL to use the dashboard
         It's the dashboard URL from the configured instance
         """
         # TODO change when instance_name can be a name instead of just a URL
         return self.instance_name
 
+    @property
+    def workspace_dashboard_url(self) -> str:
+        """
+        The dashboard URL to use the dashboard with workspace information
+        It's the dashboard URL from the configured instance
+        """
+        account = self.auth_config.get_instance(self._cmdline_instance_name).account
```

Comment:
I think this should use `self.instance_name` instead of `self._cmdline_instance_name`, and then you don't need the change in `authconfig.get_instance()`. But don't spend time on this if we end up not adding this property as we discussed.